site/.vitepress/sidebar/packages/aa-alchemy.ts

```diff
@@ -32,6 +32,17 @@ export const aaAlchemySidebar: DefaultTheme.SidebarItem = {
       text: "Light Account Client",
       link: "/light-account-client/",
     },
+    {
+      text: "Modular Account Client",
+      base: "/packages/aa-alchemy/modular-account-client",
+      collapsed: true,
+      items: [
+        {
+          text: "Multisig Account Client",
+          link: "/multisig-account-client",
+        },
+      ],
+    },
```

Comment:
sorry for the nit here, but can we flatten this? 
```suggestion
          text: "Multisig Account Client",
          link: "/packages/aa-alchemy/modular-account-client/multisig-account-client",
        },
```